pulsar-broker/src/test/java/org/apache/pulsar/client/api/TokenExpirationProduceConsumerTest.java

```diff
@@ -132,6 +136,29 @@ private PulsarAdmin getAdmin(String token) throws Exception {
         return clientBuilder.build();
     }
 
+    @Test
+    public void testNonPersistentTopic() throws Exception {
```

Comment:
I suggest that you move this test to the `org.apache.pulsar.client.api.TlsProducerConsumerTest`.

No need to change the pulsar configuration.

Reply:
After checking, it doesn't work. By default, 'brokerClientTls' is not enabled. I'll leave it here for now and move it later if needed.

Reply:
```java
    @Test
    public void testNonPersistentTopic() throws Exception {
        internalSetUpForNamespace();

        String topic = "non-persistent://my-property/my-ns/test-token-non-persistent";

        PulsarClient pulsarClient = pulsar.getClient();

        @Cleanup
        Consumer<byte[]> consumer = pulsarClient.newConsumer().topic(topic)
                .subscriptionInitialPosition(SubscriptionInitialPosition.Earliest)
                .subscriptionName("test").subscribe();

        @Cleanup
        Producer<byte[]> producer = pulsarClient.newProducer().topic(topic).create();
        byte[] msg = "Hello".getBytes(StandardCharsets.UTF_8);
        producer.send(msg);

        Message<byte[]> receive = consumer.receive(3, TimeUnit.SECONDS);
        assertNotNull(receive);
        assertEquals(receive.getData(), msg);
    }
```